### DISCUSSION TITLE
act CLI not able to download default image

### DISCUSSION THREAD

**Discussion by Thunder80** [nektos/act#2191] [Q&A] [answered]
I am just installed act CLI with brew. The commands like `act --version`, `act -l` are working but when I try to run `act --container-architecture linux/amd64` I get 

<img width="1077" alt="Screenshot 2024-02-01 at 5 12 01 PM" src="https://github.com/nektos/act/assets/43901109/e60c4ab6-4f29-442f-a887-c886f124ebfd">

OS: MacOS(with M1 chip)
Docker: 24.0.7

Any help?

**Comment by tmknight** [comment, marked as answer]
I find this to be fixed with release [0.2.59](https://github.com/nektos/act/releases/tag/v0.2.59)
